websites/R/RhysBOT/presence.ts

```diff
@@ -0,0 +1,38 @@
+const presence = new Presence({
+		clientId: "658765364439810048"
+	}),
+	date = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "Logo",
+			details: "Exploring",
+			startTimestamp: date,
+			buttons: [
+				{
+					label: "View Page",
+					url: document.location.href
+				}
+			]
+		},
+		page = document.location.pathname;
+
+	if (document.querySelector("#itemName")) {
+		presenceData.largeImageKey =
+			document.querySelector<HTMLImageElement>("#itemImage").src;
+
+		presenceData.details = `Viewing ${
+			document.querySelector("#itemName").textContent
+		}`;
+
+		if (document.querySelector("#itemDescription")) {
+			presenceData.state = `${
+				document.querySelector("#itemDescription").textContent
+			}`;
+		}
+	} else if (page && !page.includes("_"))
+		presenceData.details = `Exploring the ${page} page`;
```

Comment:
you should never put a pathname in the presenceData details like this. Please add checks for each individual page and customized messages